spritecontrol.cpp

```diff
@@ -24,7 +24,7 @@ std::vector<Sprite *> SpriteGenerator::make(unsigned int n) const {
 
 	for (float y = -1.2f; y < 1.2f; y += 1.0f / sqrt(cfg.at(SpriteCount))) {
 		for (float x = -1.2f; x < 1.2f; x += 1.0f / sqrt(cfg.at(SpriteCount))) {
-			if (Noise::random() < 0.001f) {
```

Comment:
I'd prefer if this didn't change at all, but it's such a small difference I might not care. I'll figure out when I test it later.

Reply:
I only increased it mainly because there's more of them but it shouldn't matter so I've reverted it.